src/components/interaction-designer/block-editors/AdvancedExitsBuilder.vue

```diff
@@ -4,17 +4,20 @@
 
     <template v-for="(exit, i) in block.exits">
       <advanced-exit-editor
-        ref="exitEditors"
         v-if="!exit.default"
+        ref="exitEditors"
         :key="exit.uuid"
         :block="block"
         :exit="exit"
         :label="(i + 1).toString()"
-        class="advanced-block-exit-builder-item mb-3" />
+        class="advanced-block-exit-builder-item mb-3"
+        @afterExitTestChanged="handleExitChanged(exit, /* Subtracting two to account for default exit */i === block.exits.length - 2)"
+        @afterExitNameChanged="handleExitChanged(exit, /* Subtracting two to account for default exit */i === block.exits.length - 2)" />
```

Comment:
I'd suggest we pass `i` instead of condition here, then we make a test inside the `handleExitChanged(exit, i)` method.
That would be easier to read I think.

Reply:
Ya, I'm with you! Done 🧨